src/Uno.UI.RemoteControl.Host/RemoteControlServer.cs

```diff
@@ -2,22 +2,22 @@
 using System.Collections.Generic;
 using System.IO;
 using System.Net.WebSockets;
+using System.Runtime.Loader;
 using System.Threading;
 using System.Threading.Tasks;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
 using Newtonsoft.Json;
+using Uno.Extensions;
 using Uno.UI.RemoteControl.Helpers;
 using Uno.UI.RemoteControl.HotReload.Messages;
-using Microsoft.Extensions.Logging;
-using Uno.Extensions;
 using Uno.UI.RemoteControl.Messages;
-using System.Runtime.Loader;
-using Microsoft.Extensions.Configuration;
 
 namespace Uno.UI.RemoteControl.Host
 {
 	internal class RemoteControlServer : IRemoteControlServer, IDisposable
 	{
-		private readonly Dictionary<string, IServerProcessor> _processors = new Dictionary<string, IServerProcessor>();
+		////private readonly Dictionary<string, IServerProcessor> _processors = new();
```

Comment:
To avoid the static instance, can't we just have an instance of the SharedProcessorPool tracked here, instead of the Dictionary?

Reply:
moved to the static field version

Reply:
The point of making it static was to enable a shared processor pool so that it's possible to have messages sent from one client relayed to others.

In terms of race conditions, I cannot foresee any situations where a message not being relayed to all clients is a problem, as this should only be an issue when connections are first being made, and this is when important messages are least likely to be sent. Any subsequent messages can always be resent if obviously not received by another client.